recipes/pyre/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -euo pipefail
+
+if [ `uname` = "Darwin" ]; then
+    # Use specified macOS SDK, and enforce minimum version
+    export CXXFLAGS="$CXXFLAGS -isysroot $CONDA_BUILD_SYSROOT"
+else
+    # Remove after resolving https://github.com/pyre/pyre/issues/64
+    export LDFLAGS="-lrt $LDFLAGS"
+fi
+
+mkdir build && cd build
+
+# Compute sitepackage dir relative to install prefix
```

Comment:
There is an environment variable for this $SP_DIR, https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#env-vars

Reply:
The cmake scripts internally expect the python module dir to be relative to CMAKE_INSTALL_PREFIX. I'm fixing this assumption in an open PR on pyre, but until then I have to compute $SP_DIR relative to $PREFIX, unless there's a relative equivalent of $SP_DIR I'm not aware of?

Reply:
Well, you could take $SP_DIR and chop the len($PREFIX) characters off the front of it.

Reply:
This seems more fragile imo because it depends on SP_DIR being a subdirectory of PREFIX, and breaks depending on whether PREFIX has a trailing '/'. But hopefully we won't need this workaround for long.